BUILD

```diff
@@ -28,6 +29,13 @@ multirun(
     visibility = ["//visibility:public"],
 )
 
+shellcheck(
+    scrs = glob([
```

Comment:
This doesn't work, because you cannot access files in other packages using `glob`. I.e. this glob will only return the script files at the root package. You need to add the shellcheck rule to every package that has script files.